src/CodeBlock/CodeBlock.vue

```diff
@@ -397,7 +400,12 @@ span {
 
 <script>
 import Prism from "prismjs";
-import "prismjs/components/prism-go.js";
+import "prismjs/components/prism-go.min.js";
+import "prismjs/components/prism-rust.min.js";
+import "prismjs/components/prism-markdown.min.js";
+import "prismjs/components/prism-bash.min.js";
+import "prismjs/components/prism-json.min.js";
+import "prismjs/components/prism-protobuf.min.js";
```

Comment:
These languages need to be imported manually. https://prismjs.com/#supported-languages

Reply:
Default languages:
![image](https://user-images.githubusercontent.com/1021101/86039609-86550180-b9f7-11ea-9675-f47f47dc34a9.png)

Reply:
I meant when this component is used in an app, do all of these languages get imported too? Because if that's the case, might be a bit of a heavy component. Not sure what the alternative is…